intercepting-filter/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.iluwatar</groupId>
+	<artifactId>java-design-patterns</artifactId>
```

Comment:
Did you copy the parent pom.xml here? This is not correct. Please see how the other modules have defined their pom.xml. Also, you need to add intercepting-filter as module to parent pom.xml.